common/djangoapps/pipeline_js/templates/xmodule.js

```diff
@@ -4,21 +4,20 @@
 ## These files assume that several libraries are available and bound to
 ## variables in the global context, so we load those libraries with requirejs
 ## and attach them to the global context manually.
-define(["jquery", "underscore", "mathjax", "codemirror", "tinymce",
+define(["jquery", "underscore", "codemirror", "tinymce",
         "jquery.tinymce", "jquery.qtip", "jquery.scrollTo", "jquery.flot",
         "jquery.cookie",
         "utility"],
-       function($, _, MathJax, CodeMirror, tinymce) {
+       function($, _, CodeMirror, tinymce) {
     window.$ = $;
     window._ = _;
-    window.MathJax = MathJax;
+    require(['mathjax']);
```

Comment:
This makes the loading of MathJax asynchronous which is great. However, this now mean that any code that looks at window.MathJax has to have fallback logic in case it hasn't been defined yet. I remember at least one PR that did something like this, so do we believe that it is all good now? 

I think you should add a unit test that simulates MathJax not being available and verifies that the xmodules still render in some reasonable fashion.